src/control/controlobjectscript.h

```diff
@@ -22,14 +22,14 @@ class ControlObjectScript : public ControlProxy {
             return m_scriptConnections.first(); };
     void disconnectAllConnectionsToFunction(const QJSValue& function);
 
-    // Called from update();
-    void emitValueChanged() override {
-        emit trigger(get(), this);
+    /// Called from `ControlEngineScriptLegacy::trigger`
```

Comment:
Please remove this comment then, because it is confusing. It's a public method that is accessible by anyone.